controllers/ExportController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\legal\controllers;
+
+use humhub\modules\legal\Module;
+use humhub\modules\legal\services\ExportService;
+use humhub\modules\user\components\BaseAccountController;
+use Yii;
+use yii\web\BadRequestHttpException;
+
+/* @property Module $module */
+class ExportController extends BaseAccountController

```

Comment:
Can we restrict the controller access against `isEnabledExportUserData`?

I know there is an additional check in the Service. But better protect the controller as well.

Reply:
> Can we restrict the controller access against `isEnabledExportUserData`?

@luke- Commit https://github.com/humhub/legal/pull/77/commits/95e7428241a42f4095be916f18746a90c9d24254.